### PR TITLE
Add Elastic Email plugin

### PR DESCRIPTION
**Pull request by narendravaghela**
https://github.com/pnglabz/cakephp-elastic-email

### CODE REVIEW

**Comment by ADmad** [comment]
The readme under "Usage" section it says "Mailgun" :)

**Comment by narendravaghela** [comment]
Oops! Fixed it :)

File: README.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ Additional lists you might find useful:
 ## Email
 *Plugins for sending and parsing email.*
 
+- [Elastic Email plugin](https://github.com/pnglabz/cakephp-elastic-email) - Email transport plugin for sending email via Elastic Email API.
```
Comment (ADmad) [inline README.md]: Github URL seems to have changed now.

Comment (narendravaghela) [reply README.md]: Yes, the organization name is changed. Let me update.

**Comment by narendravaghela** [comment]
The new URL - https://github.com/sprintcube/cakephp-elastic-email
I have updated the readme file as well.

**Comment by narendravaghela** [comment]
Thanks @ADmad

**Pull request merged**